Monika After Story/game/script-farewells.rpy

```diff
@@ -947,6 +947,25 @@ label bye_going_somewhere:
         $ _history_list.pop()
         m 1lksdlc "..."
         m 1eksdlb "What am I saying? Of course I'll go with you!"
+        if mas_isMoniLove(higher=True):
+        m 1hub "Oh, okay!"
+        menu:
+            m 3tub "Where are you taking me?"
+            "School.":
+                m 1eka "Ahaha [Player], I cant wait to see all your friends!"
+            "Work.":
+                m 1eka "How Fun!"
+            "A date.":
+                m 1eka "How Romantic [Player]!"
+            "Vacation.":
+                m 1eka "Im sure its going to be amazing"
+            "Beach.":
+                m 1eka "Want to see me in a bikini eh?
+                m 2hub "Ok lets go [Player]!"
+            "Road Trip.":
+                m 1eka "You Remembered. How Romantic!"
```

Comment:
You don't have to capitalize "Remembered" or "Romantic". 

I would also replace that period with a comma. 

Also, are you talking about the topic where Monika mentions she would want to go on a road trip? There are a few issues with this. Not everyone is going to have seen that topic, I find it a bit of a strange thing to be happy that the player remembered, and Monika probably meant that she wanted to go on a road trip after she crosses over, otherwise, she won't really be able to see anything.